packages/dx-react-chart/docs/reference/argument-axis.md

```diff
@@ -28,7 +28,7 @@ position? | 'bottom' &#124; 'top' | 'bottom' | The axis position.
 name? | string | | The axis name.
 indentFromAxis? | number | 10 | The indent from the axis.
 type? | 'band' &#124; 'linear' | 'linear' | Axis type.
-tickForamt? | (scale: object) => (tick: string) => string  | | A function that returns a tick formatter function.
+tickFormat? | (scale: object) => (tick: string) => string  | | A function that returns a tick formatter function.
```

Comment:
Is there an interface for the `scale` object? Or it accepts any obj?

Reply:
There is any object. Maybe we should  concretize what fields in this object can be.